<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frm_UserManagement</class>
 <widget class="QWidget" name="Frm_UserManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统用户管理</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>811</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>用户信息编辑     当前状态：</string>
   </property>
   <widget class="QLabel" name="lbl_status">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>新增用户</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Save">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>152</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>保  存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Clear">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>152</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>清  空</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Address">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>89</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Department">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_UPassword">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>58</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_UPosition">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>58</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>密      码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>32</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>单位部门：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>91</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>联系地址：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>职务职位：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>91</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>用户角色：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>121</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>备注：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>联系电话：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>32</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>32</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>真实姓名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_TrueName">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_UserName">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Telephone">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>58</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_URemark">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>119</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="txt_URole">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>89</y>
      <width>180</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>121</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>用户状态：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="txt_UStatus">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>119</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>715</x>
      <y>32</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>(*)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>253</x>
      <y>33</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>(*)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>523</x>
      <y>60</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>(*)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>783</x>
      <y>60</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>(*)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>523</x>
      <y>32</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>(*)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>783</x>
      <y>91</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>(*)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Check">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>28</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>检测</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>783</x>
      <y>121</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>(*)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_Note">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>811</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>用户信息列表</string>
   </property>
   <widget class="QPushButton" name="btn_Delete">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>删除选中用户</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tv_UserInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>791</width>
      <height>351</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ResetPwd">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>重置选中用户的密码</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txt_TrueName</tabstop>
  <tabstop>txt_Department</tabstop>
  <tabstop>txt_UserName</tabstop>
  <tabstop>btn_Check</tabstop>
  <tabstop>txt_UPosition</tabstop>
  <tabstop>txt_Telephone</tabstop>
  <tabstop>txt_UPassword</tabstop>
  <tabstop>txt_Address</tabstop>
  <tabstop>txt_URole</tabstop>
  <tabstop>txt_URemark</tabstop>
  <tabstop>txt_UStatus</tabstop>
  <tabstop>btn_Save</tabstop>
  <tabstop>btn_Clear</tabstop>
  <tabstop>tv_UserInfo</tabstop>
  <tabstop>btn_Delete</tabstop>
  <tabstop>btn_ResetPwd</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
